### PR TITLE
[ZEPPELIN-3910] IPython delete temp file and close stream

### PR DESCRIPTION
**Pull request by yx91490**
### What is this PR for?
delete temporary file  and  close inputstream in IPythonInterpreter.checkIPythonPrerequisite()

### What type of PR is it?
[Bug Fix]

### Todos
*no

### What is the Jira issue?
* https://jira.apache.org/jira/browse/ZEPPELIN-3910, [ZEPPELIN-3910]

### How should this be tested?
* manual.

### Questions:
* Does the licenses files need update?
no
* Is there breaking changes for older versions?
no
* Does this needs documentation?
no

### CODE REVIEW

File: python/src/main/java/org/apache/zeppelin/python/IPythonInterpreter.java [hunk 1]
```diff
@@ -173,26 +175,31 @@ public String checkIPythonPrerequisite(String pythonExec) {
...
-      }
-      if (!freezeOutput.contains("ipython=")) {
-        return "ipython is not installed";
-      }
-      if (!freezeOutput.contains("grpcio=")) {
-        return "grpcio is not installed";
-      }
-      if (!freezeOutput.contains("protobuf=")) {
-        return "protobuf is not installed";
+      try (FileInputStream in = new FileInputStream(stdoutFile)) {
```
[the remark this answers is not in the document]
Comment (yx91490) [reply python/src/main/java/org/apache/zeppelin/python/IPythonInterpreter.java]: It's jdk7's try-with-resources feature, which will call FileInputStream.close() finally whether occur an exception or not.

[the remark this answers is not in the document]
Comment (felixcheung) [reply python/src/main/java/org/apache/zeppelin/python/IPythonInterpreter.java]: got it

**Comment by felixcheung** [comment]
can you check failed tests https://travis-ci.org/yx91490/zeppelin/builds/468316435

**Comment by yx91490** [comment]
> can you check failed tests https://travis-ci.org/yx91490/zeppelin/builds/468316435

I rebuild it and all test passed.

**Comment by zjffdu** [comment]
LGTM

**Comment by zjffdu** [comment]
@yx91490 This PR is mixed with other commits.

**Comment by felixcheung** [comment]
you can rewrite log and force push - or it might be easier to open a new PR

**Pull request closed without merging**